presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -470,17 +470,18 @@ Node Scheduler Properties
     across all worker nodes. Setting it too high may increase query
     latency and increase CPU usage on the coordinator.
 
-``node-scheduler.network-topology``
+``node-scheduler.policy``
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
     * **Type:** ``string``
-    * **Allowed values:** ``legacy``, ``flat``
-    * **Default value:** ``legacy``
+    * **Allowed values:** ``uniform``, ``topology``
```

Comment:
I think `uniform` is better since that describes the behavior of the scheduler. `colocated ` is a specific use case, but there may be other reasons to use that scheduler policy.

Maybe we could update the documentation to include the word "colocated" in case someone searches for that?